community/teams/cuda.tt

```diff
@@ -0,0 +1,127 @@
+[% WRAPPER layout.tt title="CUDA Team" handlesLayout=1 %]
+
+<div class="page-title">
+  <div>
+    <a href="[% root%]community/index.html#governance-teams">Community</a>
+    <span>→</span>
+  </div>
+  <h1>CUDA Team</h1>
+</div>
+
+<section class="governance-description">
+  The CUDA team is responsible for everything related to
+  <a href="https://developer.nvidia.com/cuda-zone">CUDA</a> in
+  <a href="https://github.com/NixOS/nixpkgs">Nixpkgs</a>.
+</section>
+
+<section class="governance-team">
+  <aside>
+    <h2>Members</h2>
+    <ul>
+      <li>
+        Samuel A. (<a href="https://discourse.nixos.org/u/samuela">@samuela</a
+        >)<br />
+        CUDA team creator
+      </li>
+      <li>
+        Serge K. (<a href="https://discourse.nixos.org/u/sergek">@sergek</a
+        >)<br />
+        CUDA maintainer
+      </li>
+      <li>
+        Connor Baker (<a href="https://discourse.nixos.org/u/connorbaker"
+          >@connorbaker</a
+        >)<br />
+        CUDA maintainer, sponsored by <a href="https://tweag.io">Tweag</a>
+      </li>
+    </ul>
+
+    <h2>Contact</h2>
+    <ul>
+      <li>
+        <a
+          href="https://discourse.nixos.org/t/announcing-the-nixos-cuda-maintainers-team-and-a-call-for-maintainers/18074"
+          >Discourse</a
+        >
+      </li>
+      <li>
+        <a href="https://app.element.io/#/room/#cuda:nixos.org">Matrix</a>
+      </li>
+    </ul>
+  </aside>
+
+  <h2>Motivation</h2>
+  <p>
+    GPU-acceleration has become a fundamental prerequisite to machine learning
+    work and scientific computing. Without support for GPU-acceleration, the
+    Nixpkgs ecosystem will remain largely invisible to practitioners.
+  </p>
+  <p>
+    Nixpkgs and NixOS give us the tools we need to build reproducible and robust
+    packages and environments. However, the state of scientific computing in
+    Nixpkgs does not reflect this.
+  </p>
+
+  <h2>Goals and Responsibilities</h2>
+  <p>
+    The singular focus of the CUDA-maintainers team is to make Nixpkgs the best
+    resource for packages leveraging CUDA, especially in machine learning and
```

Comment:
Nit: what does it mean to be "the best resource"? Books, manuals, and forums are also "resources", are comparing nixpkgs to a book? Maybe "deployment solution", so we limit the comparison to spack, docker, conda, pip, etc